scripts/scil_compute_local_tracking.py

```diff
@@ -164,13 +204,78 @@ def tqdm_if_verbose(generator: Iterable, verbose: bool, *args, **kwargs):
     return generator
 
 
+def _save_tractogram(streamlines_generator, odf_sh_img, total_nb_seeds, args):
+    voxel_size = odf_sh_img.header.get_zooms()[0]
+
+    # wrapper for tracker.track() yielding one TractogramItem per
+    # streamline for use with LazyTractogram.
+    def tracks_generator_wrapper():
+        for strl, seed in tqdm_if_verbose(streamlines_generator,
+                                          verbose=args.verbose,
+                                          total=total_nb_seeds,
+                                          miniters=int(total_nb_seeds / 100)):
+            # seeds are returned with origin `center`
```

Comment:
This section is still confusing for me.  Can you start the whole section with this sentence:

```
Note: 
   -> Tracking is done in voxel space, center origin.  
   -> LazyTractogram seem to expects data in world coordinates, corner origin, although this is not specified anywhere.
   -> Saving seeds requires origin center from our own convention. Our other scripts such as scil_compute_seed_density_map expect so.
```